spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -375,7 +373,12 @@ def main(argv=None):
         data_out = sct_math.denoise_nlmeans(data, patch_radius=p, block_radius=b)
 
     elif arguments.symmetrize is not None:
-        data_out = (data + data[list(range(data.shape[0] - 1, -1, -1)), :, :]) / float(2)
+        if arguments.symmetrize == 0:
+            data_out = (data + data[::-1, :, :]) / float(2)
+        elif arguments.symmetrize == 1:
+            data_out = (data + data[:, ::-1, :]) / float(2)
+        elif arguments.symmetrize == 2:
+            data_out = (data + data[:, :, ::-1]) / float(2)
```

Comment:
This is just a nitpick (since the function is already quite small anyway) but I think we can simplify the logic a little by using [`numpy.flip`](https://numpy.org/doc/stable/reference/generated/numpy.flip.html)? (I also swapped out `float(2)` for `2.0`.)

```python
    data_out = (data + np.flip(data, axis=arguments.symmetrize)) / 2.0
```

I think this is basically the same idea, but perhaps a little more readable?

Reply:
The exact code would change if we made this a function in `math.py` of course!